src/test/regress/citus_tests/arbitrary_configs/citus_arbitrary_configs.py

```diff
@@ -151,14 +149,19 @@ def copy_test_files_with_names(test_names, sql_dir_path, expected_dir_path, conf
             continue
 
         sql_name = os.path.join("./sql", test_name + ".sql")
-        output_name = os.path.join("./expected", test_name + ".out")
-
         shutil.copy(sql_name, sql_dir_path)
-        if os.path.isfile(output_name):
+
+        output_name = os.path.join("./expected", test_name + ".out")
+        alt_output_version_no = 0
+        while os.path.isfile(output_name):
```

Comment:
Please add a comment on why we do this i.e. significance of multiple output files and the need to check them.

Reply:
This will help new-comers.